snaps/openvino/snap/snapcraft.yaml

```diff
@@ -12,16 +12,41 @@ description: |
 grade: devel
 confinement: strict
 
+slots:
+  intel-npu-slot:
```

Comment:
https://snapcraft.io/docs/custom-device-interface
I read this:
```
To prevent connection to arbitrary custom-device slots, the plug and slot must share the same custom-device attributes, including the name of the plug or slot:
```
What do you think ?

Reply:
Right - that's what the `custom-device: intel-npu-device` is doing below (you should see that line in both the slot and plug definitions).

Reply:
Yeah I tried that originally but the pack step fails with:

```
~/intel-neural-processing-unit/snaps/vpu-umd-test$ snapcraft 
Generated snap metadata                                                                                               
Cannot pack snap: error: cannot validate snap "vpu-umd-test": cannot have plug and slot with the same name: "intel-npu"
```

That's with the following changes:

```diff
 slots:
-  intel-npu-slot:
+  intel-npu:
 
 plugs:
-  intel-npu-plug:
+  intel-npu:
     interface: custom-device
     custom-device: intel-npu-device
 
@@ -37,7 +43,7 @@ apps:
 
   vpu-umd-test:
     command: usr/bin/vpu-umd-test
-    plugs: [intel-npu-plug]
+    plugs: [intel-npu]

```

I'm not sure if the docs are just out of date, or what.